components/layout/index.coffee

```diff
@@ -38,6 +38,26 @@ $ ->
 
   $('body').on 'click', '.js-language-toggle', toggleLanguageOnClick
 
-
+  $('body').on 'click', '.header-section-nav a', (e)->
+    section = $(e.currentTarget).data 'section'
+    $('html,body').animate { scrollTop: $("##{section}").offset().top - 140 }, 1000
+
+  # headerImageLink = "./images/Banner_"
+  incrementer = 1
+
+  changeHeaderBackground = ->
+    # $('.header').animate { 'background-image': 'url(' + headerImageLink + (incrementer++) + '.jpg)' }, 1000
```

Comment:
Better to leave commented out code removed. With version control there's usually no need to keep old code around.